### PR TITLE
Add first draft of admin guide

### PR DESCRIPTION
**Pull request by umar-ahmed**
For use in the Collab.Land help guide: https://help.collab.land/marketplace/

### CODE REVIEW

File: docs/admin-guide.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Your community members can then start using `/chainpatrol check` to start
+checking URLs whenever they are unsure of a message that has been sent in your
+Discord server. If they find a scam, they can use the `/chainpatrol report`
+command to report the scam to ChainPatrol.
+
+### Configuration
+
+Without configuration, reports will be sent to the ChainPatrol team for review.
+If you would like to be able to review reports and keep note of which community
+members are helping protect your server, you can reach out to the ChainPatrol
```
Comment (iSpeakNerd) [inline docs/admin-guide.md]: where do they reach out? twitter, email, etc?

File: docs/admin-guide.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### Getting Started
+
+To get started, install the ChainPatrol mini-app from the
+[Collab.Land Marketplace](https://cc.collab.land/).
+
+Your community members can then start using `/chainpatrol check` to start
+checking URLs whenever they are unsure of a message that has been sent in your
+Discord server. If they find a scam, they can use the `/chainpatrol report`
+command to report the scam to ChainPatrol.
```
Comment (iSpeakNerd) [inline docs/admin-guide.md]: does this mean that each community benefits from the reports made by other communities? i.e. global blocklist?

File: docs/admin-guide.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+on your Discord server. Use the `/chainpatrol <sub-command>` commands to empower
+your community to protect themselves from phishing attacks and other scams.
+
+### Getting Started
+
+To get started, install the ChainPatrol mini-app from the
+[Collab.Land Marketplace](https://cc.collab.land/).
+
+Your community members can then start using `/chainpatrol check` to start
+checking URLs whenever they are unsure of a message that has been sent in your
```
Comment (iSpeakNerd) [inline docs/admin-guide.md]: > Your community members can then start using `/chainpatrol check` to start
checking URLs whenever they are unsure of a message that has been sent in your
Discord server

Your community members can then use `/chainpatrol check` to check suspect URLs against the <server/global?> scam list when a suspicious message is sent in your Discord server.

File: docs/admin-guide.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Without configuration, reports will be sent to the ChainPatrol team for review.
+If you would like to be able to review reports and keep note of which community
+members are helping protect your server, you can reach out to the ChainPatrol
+team to get a custom admin dashboard associated to your server.
+
+---
+
+<center>
+
+[Get access to admin dashboard](https://chainpatrol.io/beta)
```
Comment (iSpeakNerd) [inline docs/admin-guide.md]: is this required or optional?

**Review by iSpeakNerd — COMMENTED** [review]
comments made in a few places

File: docs/admin-guide.md (hunk 3 above)
Comment (iSpeakNerd) [reply docs/admin-guide.md]: Report scam URLs using the `/chainpatrol report` command to report the scam to ChainPatrol.

File: docs/admin-guide.md (hunk 2 above)
Comment (umar-ahmed) [reply docs/admin-guide.md]: Yup, it's a shared blocklist for all users, not only in Discord, but also Coinbase and Metamask use the data to protect their users as well.

File: docs/admin-guide.md (hunk 1 above)
Comment (umar-ahmed) [reply docs/admin-guide.md]: Added a link to draft an email to us to get access. Also added email, twitter, and website links at the end

File: docs/admin-guide.md (hunk 4 above)
Comment (umar-ahmed) [reply docs/admin-guide.md]: Optional, added in brackets to the subheading

**Pull request merged**